docs/installation_configuration_guide/Common.md

```diff
@@ -19,11 +19,15 @@ Nussknacker configuration is divided into several configuration areas, each area
 
 * [Designer](/about/GLOSSARY#nussknacker-designer) configuration (web application ports, security, various UI settings, database),
 * Scenario Types configuration, comprising of:
-  * [Deployment Manager](/about/GLOSSARY#deployment-manager) configuration, 
   * [Model](/about/GLOSSARY#model) configuration.
-  * [Category](/installation_configuration_guide/DesignerConfiguration/#scenario-type-categories) configuration
+  * [Deployment](/about/GLOSSARY#deployment-manager) configuration,
+  * [Category](./DesignerConfiguration.md/#scenario-type-categories) configuration
 
-The Scenario Type is a convenient umbrella term for a particular Deployment Manager configuration and the associated model configuration. Diagram below presents main relationships between configuration areas.
+[Model](/about/GLOSSARY#model) configuration defines which components and which [Processing Mode](/about/ProcessingModes) will be available for the user. 
+[Deployment](/about/GLOSSARY#deployment-manager) configuration defines how scenario using these components will be deployed on the [Engine](/about/engine).
+[Category](./DesignerConfiguration.md/#scenario-type-categories) defines who has access to the given setup.
```

Comment:
Given only the docs, it's not 100% clear to me what is a setup. Maybe we should introduce a term "engine setup"?

Reply:
Changed to
> defines who has access to the given combination of Model and Scenario Deployment